LeaderboardBackend/Authorization/UserTypeAuthorizationHandler.cs

```diff
@@ -34,10 +34,6 @@ UserTypeRequirement requirement
             {
                 context.Succeed(requirement);
             }
-            else
-            {
-                context.Fail();
-            }
```

Comment:
Why don't we want to explicitly fail here? Doesn't seem to affect auth behaviour.

Reply:
You don't need to call fail.